lib/view/users_list_page/users_list_timeline_page.dart

```diff
@@ -24,6 +26,26 @@ class UsersListTimelinePage extends ConsumerWidget implements AutoRouteWrapper {
       appBar: AppBar(
         title: Text(list.name ?? ""),
         actions: [
+          if (list.isPublic!) ...[
```

Comment:
```suggestion
          if (list.isPublic ?? false) ...[
```

古いMisskeyでは `list.isPublic` は `null` になることがあります
そのような場合に `!` を使うとこのページ自体が開けなくなるので `??` を使ったほうが安全です